src/sentry/static/sentry/app/utils/discover/styles.tsx

```diff
@@ -16,8 +16,8 @@ export const VersionContainer = styled('div')`
   display: flex;
 `;
 
-export const NumberContainer = styled('div')`
-  text-align: right;
+export const NumberContainer = styled('div')<{alignNumbers?: boolean}>`
+  text-align: ${p => (p.alignNumbers === false ? 'left' : 'right')};
```

Comment:
Instead of a boolean, should the property be `align` and take a `left | right` value?

Reply:
Ditto. I like it as words more as it's more explicit than align as a bool.